### ISSUE TITLE
Searching by Phone number not working

### ISSUE THREAD

**Issue by Arazzo** [nextcloud/contacts#1501]
### Steps to reproduce
1. Docker Nextcloud Setup with MariaDB
2. Open Contacts, Search Contacts (right search field)
3. Open Contacts, use other Search Field (on the left)

### Expected behaviour
Search for Contacts by Phone Number (or extracts)

### Actual behaviour
No Results

### Server configuration

**Operating system**:
Linux / Docker

**Web server:**
Apache

**Database:**
MariaDB

**PHP version:**
7.3.15

**Nextcloud version:** (see Nextcloud admin page)
18.0.1

**Contacts version:** (see Nextcloud apps page)
3.2.0

**Updated from an older Nextcloud or fresh install:**
fresh install

**Signing status:**

```
No errors have been found.
```

**List of activated apps:**

```

Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - admin_audit: 1.8.0
  - apporder: 0.9.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.2
  - checksum: 0.4.4
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.2.0
  - dav: 1.14.0
  - event_update_notification: 1.0.1
  - extract: 1.2.3
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_accesscontrol: 1.8.1
  - files_mindmap: 0.0.17
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - metadata: 0.11.1
  - nextbackup: 19.12.1
  - nextcloud_announcements: 1.7.0
  - notes: 3.1.5
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - password_policy: 1.8.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - scanner: 0.1.8
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - support: 1.1.0
  - systemtags: 1.8.0
  - tasks: 0.11.3
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - viewer: 1.2.0
  - workflow_pdf_converter: 1.3.1
  - workflow_script: 1.3.1
  - workflowengine: 2.0.0
Disabled:
  - encryption
  - files_automatedtagging
  - files_external
  - mail
  - photos
  - survey_client
  - user_ldap

```

**Nextcloud configuration:**

```
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***",
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "18.0.1.3",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "ssl",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "0": "mysql.utf8mb4 => true",
        "mysql.utf8mb4": true
    },
    "apps": {
        "accessibility": {
            "enabled": "yes",
            "installed_version": "1.4.0",
            "types": ""
        },
        "activity": {
            "enabled": "yes",
            "installed_version": "2.11.0",
            "types": "filesystem"
        },
        "admin_audit": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "logging"
        },
        "analytics": {
            "enabled": "no",
            "installed_version": "2.0.0",
            "types": ""
        },
        "apporder": {
            "enabled": "yes",
            "installed_version": "0.9.0",
            "types": ""
        },
        "backgroundjob": {
            "lastjob": "387"
        },
        "bookmarks": {
            "enabled": "no",
            "installed_version": "2.3.4",
            "types": ""
        },
        "bruteforcesettings": {
            "enabled": "yes",
            "installed_version": "1.5.0",
            "types": ""
        },
        "calendar": {
            "enabled": "yes",
            "installed_version": "2.0.2",
            "types": ""
        },
        "checksum": {
            "enabled": "yes",
            "installed_version": "0.4.4",
            "types": "filesystem"
        },
        "cloud_federation_api": {
            "enabled": "yes",
            "installed_version": "1.1.0",
            "types": "filesystem"
        },
        "comments": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "logging"
        },
        "contacts": {
            "enabled": "yes",
            "installed_version": "3.2.0",
            "types": ""
        },
        "core": {
            "backgroundjobs_mode": "cron",
            "enterpriseLogoChecked": "yes",
            "installed.bundles": "[\"CoreBundle\"]",
            "installedat": "1583028564.7406",
            "lastcron": "1583372704",
            "lastupdateResult": "[]",
            "lastupdatedat": "1583372599",
            "oc.integritycheck.checker": "[]",
            "public_files": "files_sharing\/public.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "theming.variables": "8981a7f87797e80ed9e021eb5fe07f0f",
            "vendor": "nextcloud"
        },
        "dav": {
            "buildCalendarReminderIndex": "yes",
            "buildCalendarSearchIndex": "yes",
            "enabled": "yes",
            "installed_version": "1.14.0",
            "regeneratedBirthdayCalendarsForYearFix": "yes",
            "sendEventRemindersPush": "yes",
            "types": "filesystem"
        },
        "deck": {
            "enabled": "no",
            "installed_version": "0.8.0",
            "types": "dav"
        },
        "event_update_notification": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": "logging,dav"
        },
        "extract": {
            "enabled": "yes",
            "installed_version": "1.2.3",
            "types": ""
        },
        "federatedfilesharing": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": ""
        },
        "federation": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "authentication"
        },
        "files": {
            "cronjob_scan_files": "500",
            "enabled": "yes",
            "installed_version": "1.13.1",
            "types": "filesystem"
        },
        "files_accesscontrol": {
            "enabled": "yes",
            "installed_version": "1.8.1",
            "types": "filesystem"
        },
        "files_automatedtagging": {
            "enabled": "no",
            "installed_version": "1.8.0",
            "types": "filesystem"
        },
        "files_mindmap": {
            "enabled": "yes",
            "installed_version": "0.0.17",
            "types": ""
        },
        "files_pdfviewer": {
            "enabled": "yes",
            "installed_version": "1.7.0",
            "types": ""
        },
        "files_rightclick": {
            "enabled": "yes",
            "installed_version": "0.15.2",
            "types": ""
        },
        "files_sharing": {
            "enabled": "yes",
            "installed_version": "1.10.1",
            "lookupServerUploadEnabled": "no",
            "outgoing_server2server_share_enabled": "no",
            "types": "filesystem"
        },
        "files_trashbin": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "filesystem,dav"
        },
        "files_versions": {
            "enabled": "yes",
            "installed_version": "1.11.0",
            "types": "filesystem,dav"
        },
        "files_videoplayer": {
            "enabled": "yes",
            "installed_version": "1.7.0",
            "types": ""
        },
        "firstrunwizard": {
            "enabled": "yes",
            "installed_version": "2.7.0",
            "types": "logging"
        },
        "logreader": {
            "enabled": "yes",
            "installed_version": "2.3.0",
            "levels": "11111",
            "types": ""
        },
        "lookup_server_connector": {
            "enabled": "yes",
            "installed_version": "1.6.0",
            "types": "authentication"
        },
        "mail": {
            "enabled": "no",
            "installed_version": "1.1.3",
            "types": ""
        },
        "metadata": {
            "enabled": "yes",
            "installed_version": "0.11.1",
            "types": ""
        },
        "nextbackup": {
            "enabled": "yes",
            "installed_version": "19.12.1",
            "types": ""
        },
        "nextcloud_announcements": {
            "enabled": "yes",
            "installed_version": "1.7.0",
            "pub_date": "Thu, 24 Oct 2019 00:00:00 +0200",
            "types": "logging"
        },
        "notes": {
            "enabled": "yes",
            "installed_version": "3.1.5",
            "types": ""
        },
        "notifications": {
            "enabled": "yes",
            "installed_version": "2.6.0",
            "types": "logging"
        },
        "oauth2": {
            "enabled": "yes",
            "installed_version": "1.6.0",
            "types": "authentication"
        },
        "ocsms": {
            "enabled": "no",
            "installed_version": "2.1.7",
            "types": ""
        },
        "password_policy": {
            "enabled": "yes",
            "enforceHaveIBeenPwned": "1",
            "enforceNumericCharacters": "1",
            "enforceSpecialCharacters": "1",
            "enforceUpperLowerCase": "1",
            "installed_version": "1.8.0",
            "minLength": "9",
            "types": ""
        },
        "photos": {
            "enabled": "no",
            "installed_version": "1.0.0",
            "types": ""
        },
        "privacy": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": ""
        },
        "provisioning_api": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "prevent_group_restriction"
        },
        "quicknotes": {
            "enabled": "no",
            "installed_version": "0.2.3",
            "types": ""
        },
        "recommendations": {
            "enabled": "yes",
            "installed_version": "0.6.0",
            "types": ""
        },
        "scanner": {
            "enabled": "yes",
            "installed_version": "0.1.8",
            "types": ""
        },
        "serverinfo": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": ""
        },
        "settings": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": ""
        },
        "sharebymail": {
            "enabled": "yes",
            "enforcePasswordProtection": "yes",
            "installed_version": "1.8.0",
            "types": "filesystem"
        },
        "spreed": {
            "enabled": "no",
            "installed_version": "8.0.4",
            "stun_servers": "***REMOVED SENSITIVE VALUE***",
            "types": "prevent_group_restriction"
        },
        "support": {
            "enabled": "yes",
            "installed_version": "1.1.0",
            "types": "session"
        },
        "survey_client": {
            "apps": "no",
            "database": "no",
            "enabled": "no",
            "encryption": "no",
            "files_sharing": "no",
            "installed_version": "1.6.0",
            "php": "no",
            "server": "no",
            "stats": "no",
            "types": ""
        },
        "systemtags": {
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "logging"
        },
        "tasks": {
            "enabled": "yes",
            "installed_version": "0.11.3",
            "types": ""
        },
        "telephoneprovider": {
            "enabled": "no",
            "installed_version": "1.0.3",
            "types": ""
        },
        "text": {
            "enabled": "yes",
            "installed_version": "2.0.0",
            "types": "dav"
        },
        "theming": {
            "backgroundMime": "image\/jpeg",
            "cachebuster": "6",
            "color": "#21174a",
            "enabled": "yes",
            "installed_version": "1.9.0",
            "logoMime": "image\/gif",
            "name": "***REMOVED SENSITIVE VALUE***",
            "slogan": "***REMOVED SENSITIVE VALUE***",
            "types": "logging",
            "url": "***REMOVED SENSITIVE VALUE***"
        },
        "timetracker": {
            "enabled": "no",
            "installed_version": "0.0.42",
            "types": ""
        },
        "twofactor_backupcodes": {
            "enabled": "yes",
            "installed_version": "1.7.0",
            "types": ""
        },
        "updatenotification": {
            "calendar": "2.0.2",
            "enabled": "yes",
            "installed_version": "1.8.0",
            "types": "",
            "update_check_errors": "0"
        },
        "viewer": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": ""
        },
        "weather": {
            "enabled": "no",
            "installed_version": "1.7.1",
            "types": ""
        },
        "workflow_pdf_converter": {
            "enabled": "yes",
            "installed_version": "1.3.1",
            "types": "filesystem"
        },
        "workflow_script": {
            "enabled": "yes",
            "installed_version": "1.3.1",
            "types": "filesystem"
        },
        "workflowengine": {
            "enabled": "yes",
            "installed_version": "2.0.0",
            "types": "filesystem"
        },
        "xray": {
            "enabled": "no",
            "installed_version": "0.2.3",
            "types": "logging"
        }
    }
}

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
No

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No

### Client configuration
**Browser:**
73.0.1 (64-Bit)

**Operating system:**
Mac OS

**CardDAV-clients:**
None

#### Web server error log
```
```

#### Nextcloud log (data/nextcloud.log)
```
  GNU nano 3.2                                         nextcloud.log                                                   

{"reqId":"Vn3BTf2Yj3c3jQFrJ9kf","level":2,"time":"2020-03-01T02:11:01+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"Pgcyaa18sxUtoePPyTDL","level":2,"time":"2020-03-01T02:19:31+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"jWpA6GylHhlGebziEpDY","level":2,"time":"2020-03-01T02:58:32+00:00","remoteAddr":"87.44.101.127","user":"--",$
{"reqId":"iqADy2VoHypAAfit02Ne","level":2,"time":"2020-03-01T02:58:48+00:00","remoteAddr":"87.44.101.127","user":"--",$
{"reqId":"Mvmiz9ZEtMod2lAzHFgL","level":2,"time":"2020-03-01T02:59:10+00:00","remoteAddr":"87.44.101.127","user":"--",$
{"reqId":"SdN0dPVbkdomhNJYkZEc","level":2,"time":"2020-03-01T02:59:17+00:00","remoteAddr":"87.44.101.127","user":"--",$
{"reqId":"AFyGGighUSIgx97Sctb2","level":2,"time":"2020-03-01T03:00:18+00:00","remoteAddr":"87.44.101.127","user":"--",$
{"reqId":"6yhjYxvI6KEXSRKrMeQq","level":3,"time":"2020-03-01T03:16:27+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"4u9rPmai9h2zIFhXbyD3","level":3,"time":"2020-03-01T03:16:45+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"G6xX3vVRwFpaNZLP8IQm","level":3,"time":"2020-03-01T03:35:38+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"Id0vByh6necAdkvJegun","level":3,"time":"2020-03-01T05:44:46+00:00","remoteAddr":"192.1.1.6","user":"Maria$
{"reqId":"pmr7iglpr8G4aG6vsuvR","level":3,"time":"2020-03-01T05:44:46+00:00","remoteAddr":"192.1.1.6","user":"Maria$
{"reqId":"4sIcR6G8gVaHC3feOSru","level":3,"time":"2020-03-01T05:44:47+00:00","remoteAddr":"192.1.1.6","user":"Maria$
{"reqId":"U516G5Az2KVdS5QQU2GK","level":3,"time":"2020-03-01T05:44:48+00:00","remoteAddr":"192.1.1.6","user":"Maria$
{"reqId":"4itWBLS9iTYxoQfSyyyM","level":3,"time":"2020-03-01T06:12:05+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"QJkQ7l6OkyqLwHydjJX2","level":3,"time":"2020-03-01T06:12:28+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"2M5GGIjlgANVJsoeqQCf","level":2,"time":"2020-03-01T06:17:28+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"D2Pa4eeRBQaXENTI0wW9","level":3,"time":"2020-03-01T07:02:30+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"V7fHuEUNlR2GSggUj3Vh","level":3,"time":"2020-03-01T07:03:18+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"YeMeSyAHBnHHfoTJVgHv","level":3,"time":"2020-03-01T12:36:12+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"jagDlftZZG3yZo1e8ooz","level":3,"time":"2020-03-01T12:40:25+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"yXZYyTqNcM4jGHMHXpSJ","level":3,"time":"2020-03-01T12:40:29+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"vIUayPpQQyCAHdrQMJZo","level":3,"time":"2020-03-01T12:53:52+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"R80rK4BtVmaexdB5zf6m","level":3,"time":"2020-03-01T12:59:10+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"R80rK4BtVmaexdB5zf6m","level":3,"time":"2020-03-01T12:59:10+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"R80rK4BtVmaexdB5zf6m","level":3,"time":"2020-03-01T12:59:10+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"R80rK4BtVmaexdB5zf6m","level":3,"time":"2020-03-01T12:59:10+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"9X6Q7PHFoKnjVfAhwUo2","level":3,"time":"2020-03-01T12:59:28+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"VUDCdT4xLTtNEr3bh5TR","level":3,"time":"2020-03-01T12:59:33+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"BpTwGPVOpzE1W9ZBesWI","level":3,"time":"2020-03-01T12:59:35+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"5G9Fg6xbTyprv656hDhF","level":3,"time":"2020-03-01T19:50:24+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"BLCzvXBhhHbNeM5rrdiP","level":3,"time":"2020-03-02T01:37:31+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"6vJAyNB0FQq7hczfbXvW","level":2,"time":"2020-03-02T01:55:08+00:00","remoteAddr":"","user":"--","app":"dav","$
{"reqId":"6vJAyNB0FQq7hczfbXvW","level":2,"time":"2020-03-02T01:55:08+00:00","remoteAddr":"","user":"--","app":"dav","$
{"reqId":"6vJAyNB0FQq7hczfbXvW","level":2,"time":"2020-03-02T01:55:09+00:00","remoteAddr":"","user":"--","app":"dav","$
{"reqId":"t4lJJvyymWvWTlgzdYOH","level":3,"time":"2020-03-02T03:44:26+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"J72FSMijGWhk80HPVlCM","level":3,"time":"2020-03-02T05:32:57+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"6lKVVM2PCou4TQo8kvLb","level":3,"time":"2020-03-02T05:51:21+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"FrtuDIifwP5rjIe24mN2","level":3,"time":"2020-03-02T06:46:56+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"2dKoYSltqhBFo7oZNS6g","level":3,"time":"2020-03-02T13:47:51+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"9SHDpVrMu0EeXNN8yPhh","level":3,"time":"2020-03-03T03:41:00+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"4kOnvQF65ZnCRDdVLBv6","level":3,"time":"2020-03-03T03:41:57+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"4kOnvQF65ZnCRDdVLBv6","level":4,"time":"2020-03-03T03:41:57+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"9d9d0QBAhLYYrHyxbyDK","level":3,"time":"2020-03-03T03:41:58+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"9d9d0QBAhLYYrHyxbyDK","level":4,"time":"2020-03-03T03:41:58+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"b9JKzbQIxTSt7eyzyz4T","level":3,"time":"2020-03-03T11:34:01+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"RAOhYvkuSebLjE4qzM6y","level":3,"time":"2020-03-03T11:42:12+00:00","remoteAddr":"192.1.1.6","user":"Maria$
{"reqId":"i49HXv4UmRNdMQHS3G7P","level":3,"time":"2020-03-04T09:07:58+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"pdMKWNkhxBxU0vIUybgt","level":3,"time":"2020-03-04T17:17:46+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"TUgbCBeqULHc7PMQvZ5F","level":3,"time":"2020-03-04T17:42:18+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"ZiTKR49VND2M4rn6J5FM","level":3,"time":"2020-03-04T17:45:29+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"BVAIFVETsUI9hqMvE4U2","level":3,"time":"2020-03-04T18:06:58+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"8ZtAwIVe8snFNjYjL9TR","level":3,"time":"2020-03-04T22:56:38+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"5oa3kB1sSV5302K3eK5x","level":3,"time":"2020-03-04T23:27:23+00:00","remoteAddr":"192.1.1.2","user":"Maria$
{"reqId":"hmJZWOXcb8jvvVKvkqAw","level":3,"time":"2020-03-04T23:47:26+00:00","remoteAddr":"","user":"--","app":"no app$
{"reqId":"2mpR30F3bqhPlOIpAOoS","level":3,"time":"2020-03-04T23:48:31+00:00","remoteAddr":"192.1.1.2","user":"Maria$
```

#### Browser log
```
```

**Comment by skjnldsv** [comment]
Duplicate of https://github.com/nextcloud/contacts/issues/534

**Comment by Arazzo** [comment]
Problem persisting in actual version, please reopen?

**Comment by skjnldsv** [comment]
My comment says this issue is a duplicate of #534.
Please follow this other one instead

**Comment by Arazzo** [comment]
Your comment is wrong then. This is not a duplicate.

It was not `Fixed in 3.0.1`

Problem still persisting. Re-open please.
